pages/common/svn-changelist.md

```diff
@@ -0,0 +1,24 @@
+# svn changelist
+
+> Associate a changelist with a set of files.
+> More information: <http://svnbook.red-bean.com/en/1.7/svn.advanced.changelists.html>.
+
+- Add files to a [new] changelist:
+
+`svn changelist {{changelist_name}} {{path/to/file1}} {{path/to/file2}}`
+
+- Remove files from a changelist:
+
+`svn changelist --remove {{path/to/file1}} {{path/to/file2}}`
+
+- Remove the whole changelist at once:
+
+`svn changelist --remove --recursive --changelist {{changelist_name}} .`
+
+- Add a whole folder to a changelist:
+
+`svn changelist --recursive {{changelist_name}} {{path/to/folder1}}`
```

Comment:
It's just 1 folder, and we use the term `directory`:
```suggestion
`svn changelist --recursive {{changelist_name}} {{path/to/directory}}`
```

Reply:
Sorry, that was my addition.